Documentation/observability/metrics.rst

```diff
@@ -628,6 +628,26 @@ Exported Metrics
 
 Hubble metrics are exported under the ``hubble_`` Prometheus namespace.
 
+lost events
+~~~~~~~~~~~
```

Comment:
Thanks! Looks good, with one observation.

Other metrics documented below have their title formatted like this:

```suggestion
``lost-events``
~~~~~~~~~~~~~
```

They're also sorted in alphabetical order. I understand this one might be a bit apart from the rest, so up to you how you format it and where you put it.

Reply:
This difference was intentional - for other metrics the title in the docs is the name of the option to enable them. Lost events metric is not enabled explicitly, so I didn't want to make it look like it is.